### DISCUSSION TITLE
Move exceeds maximum extrusion?

### DISCUSSION THREAD

**Discussion by T9Air** [jschuh/klipper-macros#248] [Q&A] [answered]
Move exceeds maximum extrusion (1.203mm^2 vs 0.640mm^2)
See the 'max_extrude_cross_section' config option for details

How do I fix this? I am using a .4 mm nozzle, which has a cross-section of 0.12566^2

**Comment by mcavalcantib** [comment]
Didi resolved this problem?

**Comment by T9Air** [comment, marked as answer]
I increased the 'max_extrude_cross_section' config section to solve the problem. It did not have any negative side effects
